### PR TITLE
Add wrap-up email to teams

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Fixes https://github.com/srobo/tasks/issues/1150
Fixes https://github.com/srobo/tasks/issues/1147

## Blocked on:

- [ ] https://github.com/srobo/website/pull/498

### CODE REVIEW

**Comment by PeterJCLaw** [comment]
Seems good so far, I'm guessing the missing links are things which haven't been pinned down yet?

File: SR2023/2023-04-12-competition-wrapup.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Future Events
+
+We hope to run a competition next year and see you all there. We plan to open places around September and will let you know as soon as they are.
+
+In the meantime, be sure to follow us on Twitter [@StudentRobotics](https://twitter.com/studentrobotics) and [Instagram](https://www.instagram.com/student_robotics). If you haven't done so already, please sign up to [our general mailing list](https://studentrobotics.org/compete/), to be notified as soon as places are open.
+
+### Volunteering
+
+Student Robotics is powered by a collection of passionate individuals with varying skills from diverse backgrounds, many of whom are former competitors. We're all volunteers who give our spare time to make these events possible. If you have students graduating this year who may be interested in volunteering, please ask them to sign up to be a blueshirt at [srobo.org/join](https://srobo.org/join).
```
Comment (trickeydan) [inline SR2023/2023-04-12-competition-wrapup.md:64]: Teachers may be interested in volunteering too, let's not exclude them :)

**Comment by RealOrangeOne** [comment]
> I'm guessing the missing links are things which haven't been pinned down yet

Exactly. They should all be captured in the blocked items above

File: SR2023/2023-04-12-competition-wrapup.md (hunk 1 above)
Comment (RealOrangeOne) [reply SR2023/2023-04-12-competition-wrapup.md:64]: This is true, although gets complicated if they want to be an active competition-side volunteer _and_ still lead a team.

Comment (trickeydan) [reply SR2023/2023-04-12-competition-wrapup.md:64]: We already have multiple team leaders who are also volunteers. I don't think we should exclude them. We have also had supporting staff become volunteers, and the current wording here excludes anyone who isn't an 18 year old student...

Comment (antoinepetty) [reply SR2023/2023-04-12-competition-wrapup.md:64]: I agree we should word it more inclusively, two of our most crucial volunteers from SR2023 fall into this category.
I'm not sure how we'd word it, but we could direct those ones to /volunteer instead?

**Review by prophile — APPROVED** [review]

**Pull request merged**